src/clojure/nrepl/middleware/sideloader.clj

```diff
@@ -0,0 +1,100 @@
+(ns nrepl.middleware.sideloader
```

Comment:
It'd be nice to document the high-level idea here as well and add "0.7" as the version where the ns was introduced.